views/about.tmpl

```diff
@@ -1,3 +1,17 @@
+{{define "title"}}{{.PageContext.ShortName}} | About{{end}}
+
+{{define "open-graph"}}
+    <meta name="twitter:card" content="summary_large_image"/>
+    <meta name="twitter:site" content="@URY1350"/>
```

Comment:
Tempted to say we should add the twitter handle to the config file as will will want it for the twitter links etc @mstratford @LordAro what do you guys think?